### PR TITLE
Update to node 8

### PR DESCRIPTION
**Pull request by sirreal**
Node 8 is now LTS. Calypso upgrade is in progress.

Update wp-calypso accordingly.

Related: https://github.com/Automattic/wp-calypso/pull/19317 and #363

### CODE REVIEW

**Comment by astralbodies** [comment]
You may want to rebase or merge in the latest from develop as I just merged in the latest changes from the release/3.1.0 branch.

**Comment by sirreal** [comment]
👌  Thanks, rebased.

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-v6.11.5
```
Comment (sirreal) [inline .nvmrc]: What do you think about symlinking this file to point to `calypso/.nvmrc`?

Comment (astralbodies) [reply .nvmrc]: That makes sense! Is that supported on Windows?

Comment (sirreal) [reply .nvmrc]: > That makes sense! Is that supported on Windows?

I don't think so 🙁 

But…

> Note: nvm does not support Windows (see #284). Two alternatives exist, which are neither supported nor developed by us […]

☝️  https://github.com/creationix/nvm#important-notes

So maybe we don't care about Windows support for `nvm` related stuff?

Comment (astralbodies) [reply .nvmrc]: > So maybe we don't care about Windows support for nvm related stuff?

Yup that sounds accurate then if nvm isn't supported. 😄

**Comment by astralbodies** [comment]
I added a commit to move Calypso ahead to latest master so the Node 8.9.1 changes were pulled in. I think the unit test will still fail for this PR because the CircleCI changes are in a different PR. I'm running the tests locally now and will merge when they pass.

**Comment by sirreal** [comment]
> I think the unit test will still fail for this PR because the CircleCI changes are in a different PR

Maybe cherry-pick addb79d731b3896c6711acb2779c993fe88060ec and close #363

**Pull request merged**